client/app/telecom/pack/xdsl/access/diagnostic/details/pack-xdl-access-diagnostic-details.controller.js

```diff
@@ -0,0 +1,44 @@
+angular.module('managerApp').controller('XdslDiagnosticDetailsCtrl', class XdslDiagnosticDetailsCtrl {
+  constructor($scope, $rootScope, $state, $translate) {
+    this.$scope = $scope;
+    this.$rootScope = $rootScope;
+    this.$state = $state; // used in html
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.diagnostic = null;
+    this.statuses = null;
+
+    this.$scope.$on('accessDiagnosticDetails:arrived', (event, diagnostic) => {
+      this.diagnostic = diagnostic;
+      this.statuses = this.constructor.getStatuses(diagnostic || {});
+    });
+
+    this.$rootScope.$emit('accessDiagnosticDetails:get');
```

Comment:
There's no other way I suppose ?

Reply:
I tried other ways, it didn't work :disappointed: